backend/src/routes/post.ts

```diff
@@ -59,9 +68,20 @@ router.post('/:postID/unlike', async (ctx, next) => {
 	ctx.body = post;
 });
 
+//delete post by postID
+router.delete('/:postID', async (ctx, next) => {
+	const post = Post.findOneBy(mongodb.ObjectId(ctx.params.postID));
+	//get author of post
+	const author = User.findOneBy(mongodb.ObjectId((await post).author));
```

Comment:
Could you prefix the right-hand sides of these assignments with `await`? Then we wouldn't need to prefix `post` and `author` with `await` every time we use them.